pkg/archive/archive.go

```diff
@@ -405,6 +405,8 @@ func Tar(path string, compression Compression) (io.ReadCloser, error) {
 // paths are included in `options.IncludeFiles` (if non-nil) or not in `options.ExcludePatterns`.
 func TarWithOptions(srcPath string, options *TarOptions) (io.ReadCloser, error) {
 
+	srcPath = fixVolumePathPrefix(srcPath)
```

Comment:
Fine but the comment in the common codepath to explain that fixVolumePathPrefix is only needed for windows is important.